### ISSUE TITLE
Space in between words

### ISSUE THREAD

**Issue by abensur** [mozilla/pdf.js#8504]
Some strings have a space in between the words:
`ANA PA UL A D'IANNI DENOFRIO` (expected: `ANA PAULA D'IANNI DENOFRIO`)

However, in chrome's reader or adobe's, it's fine, without the extra space.

This behaviour also happens with my OS's Preview program so I'm convinced this is not a particular problem in this project.

I wonder If someone could ELI5 what's happening here so I can explain to my client what's not good with his PDF.

Configuration:
- Web browser and its version: Chrome 58.0.3029.110 (64-bit)
- Operating system and its version: OS X El Capitan
- PDF.js version: '1.8.170'
- Is an extension: no

Can't provide the pdf file here, the information is sensible.

**Comment by abensur** [comment]
I've managed to extract the correct word changing the `SPACE_FACTOR` in the worker!
changed from `0.3` to `0.5`

**Comment by Hikariii** [comment]
related to https://github.com/mozilla/pdf.js/issues/6705
